README.md

```diff
@@ -12,56 +12,112 @@
 </p><br/>
 
-<h2 align="center">No Code and Low Code Integration editor</h2>
+<h1 align="left">Kaoto - Visual Editor for <a href="https://camel.apache.org">Apache Camel</a></h2>
 
-<p align="center">
+<p align="center" style="font-size: 24px">
   <a href="#features">Features</a> •
-  <a href="#requirements">Requirements</a> •
+  <a href="#installation">Installation</a> •
   <a href="https://kaoto.io/docs/">Documentation</a> •
   <a href="#issues">Issues</a>  •
   <a href="#data-and-telemetry">Telemetry</a>
-</p>
+</p><br/>
 
-<p align="center">
-<a href="https://www.kaoto.io">Kaoto</a> is an integration editor to create and deploy workflows in a visual, low-code way, with developer-friendly features like a code editor and deployments to the cloud. Kaoto augments user productivity via <a href="https://camel.apache.org">Apache Camel</a>. It accelerates new users and helps experienced developers.
+<p align="left" style="font-size: 18px">
+<a href="https://www.kaoto.io">Kaoto</a> is a visual integration editor to create and modify <a href="https://camel.apache.org">Apache Camel</a> integrations in a low-code way. Kaoto provides developer-friendly features like access to a code editor and the ability to run locally and deploy to the cloud. Kaoto augments user productivity working with <a href="https://camel.apache.org">Apache Camel</a>.
+</p><br/>
+
+<p align="center"><img style="width: 60%; height: auto;" src="images/kaoto-editor.png" alt="a screenshot of the Kaoto visual editor showing 2 Camel routes with an open configuration panel for the first Camel route" width="100%"/></p>
 </p><br/>
 
-<p align="center"><img src="images/basicDemo.gif" alt="Create file named demo.kaoto.yaml, it opens automatically, then add a new route in embedded Kaoto UI, add a model step setBody, configure content of setBody through  property panel which opens when selecting the node and save the editor" width="100%"/></p>
+## Features
+
+<p>
+  <ul style="font-size: 18px">
+    <li>Support for Enterprise Integration Patterns</li>
+    <li>Create Camel Routes, Kamelets and Pipes</li>
+    <li>Supports OpenAPI to REST DSL code generation</li>
+    <li>Support for Beans</li>
+    <li>300+ Camel Components</li>
+    <li>200+ Kamelets</li>
+    <li>Built-in catalog of Components, Kamelets and EIPs with documentation</li>
+    <li>Run your integrations locally</li>
+  </ul>
+</p><br/>
 
-### Features
+## Installation
 
-- Edit Camel files following pattern (`*.camel.yaml` and `*.camel.yml`).
-- Edit Kamelet files following pattern (`*.kamelet.yaml` and `*.kamelet.yml`).
-- Edit Pipe files following pattern (`*.pipe.yaml`, `*.pipe.yml`, `*-pipe.yaml` and `*-pipe.yml`)
-- Allow to edit `*.yaml` and `*.yml` when opening through contextual menu.
-- Edit Kaoto (`*.kaoto.yaml` and `*.kaoto.yml`) files. Note that these filename extensions are not supported by Camel JBang and are merely still here to stay backward-compatible with Kaoto v1.
```

Comment:
the suspported patterns must be visible somewhere as it was source of a lot of questions in the past